models/annotations_models/train_random_forest_classifier_model.ipynb

```diff
@@ -426,12 +240,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 20,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Make sure there is no Ashpalt.\n",
-    "df = df[df['label'] != \"Asphalt\"]"
+    "# Downsample Water if necessary\n",
```

Comment:
Why have commented out code?

Reply:
It is a experiment to test if downsampling is needed, will make the documentation better